src/index.js

```diff
@@ -7,12 +7,14 @@ import {BrowserRouter} from 'react-router-dom'
 import { AuthProvider } from "./context/auth-context";
 import { VideoProvider } from "./context/video-context";
 import { PlaylistProvider } from "./context/playlist-context";
+import { LikeProvider } from "./context/like-context";
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
   <React.StrictMode>
+    <LikeProvider>
```

Comment:
i think  browser router should be on top of other providers

Reply:
ok will change it